aziotctl/src/internal/check/checks/certs_preloaded.rs

```diff
@@ -36,41 +39,65 @@ impl CertsPreloaded {
     ) -> Result<CheckResult> {
         let preloaded_certs = &unwrap_or_skip!(&cache.cfg.certd).preloaded_certs;
 
-        // TODO?: support returning multiple check results from a single check
```

Comment:
You might want to leave this TODO comment, since this is something the check would benefit from if it were supported.

Reply:
Done.